packages/compass-sidebar/src/components/multiple-connections/sidebar.tsx

```diff
@@ -78,15 +83,45 @@ function ConnectionErrorToastBody({
   );
 }
 
+async function copyConnectionString(connectionString: string) {
+  try {
+    await navigator.clipboard.writeText(connectionString);
+    openToast('copy-to-clipboard', {
+      title: 'Success',
+      description: 'Copied to clipboard.',
+      variant: 'success',
+      timeout: TOAST_TIMEOUT_MS,
+    });
+  } catch (err) {
```

Comment:
Do we want to include `err.message` in the description here, or otherwise log it somehow? Otherwise you can do
```suggestion
  } catch {
```